### PR TITLE
Got ES6 Travis build working + Updated Webpack & Babel

### PR DESCRIPTION
**Pull request by agmonks**
#628

### CODE REVIEW

File: src/bindings.js [hunk 1]
```diff
@@ -266,31 +266,30 @@ export class ComponentBinding extends Binding {
...
+			  let stat = this.component.static
+
+			  if (stat && stat.indexOf(propertyName) > -1) {
+				this.static[propertyName] = attribute.value
+			  } else {
+				this.observers[propertyName] = attribute.value
+			  }
+			}
+		  })
+		}
```
Comment (Leeds-eBooks) [inline src/bindings.js]: Hey, sorry for the long delay, I have more time now and want to get Rivets up to date and in active development again. I want to merge your PR but could you fix the indentation here first? Thanks so much!

File: src/bindings.js [hunk 2]
```diff
@@ -417,23 +416,22 @@ export class ComponentBinding extends Binding {
...
+		this.dependencies = []
+		this.formatterObservers = {}
+
+		this.binder = {
+		  routine: (node, value) => {
+			node.data = defined(value) ? value : ''
+		  }
+		}
+
+		this.sync = this.sync.bind(this)
```
Comment (Leeds-eBooks) [inline src/bindings.js]: Same here about indentation.

**Comment by agmonks** [comment]
Done (I hope). Sorry using a development environment where we use tabs rather than spacing and  using lazy tools!

**Comment by Leeds-eBooks** [comment]
Brilliant, thank you! Also, I might be wrong, but I don't think we're committing dist files unless we're pushing a release. Can you remove the `dist/rivets.bundled.js` file in [this commit](https://github.com/mikeric/rivets/pull/627/commits/fbda5ade8de0fddea367ed659580c958a10d179f)?

**Comment by agmonks** [comment]
done

**Comment by Leeds-eBooks** [comment]
So how do we get Travis up and running? Did you get it working? This is empty: https://travis-ci.org/mikeric/rivets

**Comment by agmonks** [comment]
Hmmm, it should just show in Travis. I wonder if it needs the project owner to set it up?

https://docs.travis-ci.com/user/getting-started/

"Note: You can only enable Travis CI builds for repositories you have admin access to."

**Comment by Leeds-eBooks** [comment]
Ah I'm not admin. @mikeric any chance you could do this?

**Comment by Leeds-eBooks** [comment]
@agmonks I am hitting [this issue](https://phabricator.babeljs.io/T7305) when trying to run `npm test`. How did you work around this? We need a polyfill for phantomjs, did you get tests passing?

**Comment by agmonks** [comment]
What output are you getting form npm test? working ok on mine and the travis build: https://travis-ci.org/agmonks/rivets/builds/150660774

**Comment by Leeds-eBooks** [comment]
Ah my mistake, silly typo in my latest changes. Sorry! All works great 😄

**Comment by agmonks** [comment]
@Leeds-eBooks @mikeric Is it work rolling rivetsjs into its own github organisation (for shared management)?

**Comment by Leeds-eBooks** [comment]
@agmonks Yes this would be good, but without admin access I don't think I can make this happen.

**Pull request merged**